sdk/instrumentation/net/http/handler.go

```diff
@@ -12,14 +12,31 @@ import (
 	"github.com/hypertrace/goagent/sdk/filter"
 	internalconfig "github.com/hypertrace/goagent/sdk/internal/config"
 	"github.com/hypertrace/goagent/sdk/internal/container"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
```

Comment:
I need to move this otel specific logic out of the sdk since this is meant to be a hypertrace sdk that can be adapted by any telemetry framework. But can do that separately.